ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -521,5 +521,10 @@ open class BadgeView: UIView, TokenizedControlInternal {
 
     // MARK: Accessibility
 
-    open override var accessibilityLabel: String? { get { return label.text } set { } }
+    private var _accessibilityLabel: String?
```

Comment:
Should this be on the datasource instead? Seems a bit odd to set the text and the accessibility label in different places.